src/service/translationService.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import axios from '@nextcloud/axios'
+import { generateOcsUrl } from '@nextcloud/router'
+import store from '../store/index.js'
+
+const checkTranslationServiceAvailability = function() {
+	/// TODO
```

Comment:
@kesselb @SebastianKrupinski Hey! I got this endpoint from Marcel, but I don't have the means to test the translation API yet. Could you please do me a favor: check out the endpoint response and adjust the function appropriately? 

Apart from that this PR is ready, all the front-end logic works

Reply:
Fixed it.